### ISSUE TITLE
vd v3.0.2 performance regression on Android

### ISSUE THREAD

**Issue by snooppr** [saulpw/visidata#2382]
I use visidata on Android (Termux, Python 3.11) to view csv/xlsx.  Everything works fine with version vd **2.11**.  Updated to the latest version vd **3.0.2**.  It became impossible to use, terrible brakes: the movement of the cursor downwards/upwards/sideways is delayed by seconds.  I tried even small files of 200 lines, it’s impossible to use at the moment, I rolled back to the previous version, everything works.

v3.0.2
![ScreenRec](https://github.com/saulpw/visidata/assets/61022210/0a86512d-7f83-461b-a928-7cc5dc224c3b)

---------

v2.11
![ScreenRecord](https://github.com/saulpw/visidata/assets/61022210/18c52e56-3189-40bf-b1b6-0589f5dc912d)

Simply put, the delay in responding to commands has increased tenfold.

In addition, in v3.0.2 in xlsx the selected line on which the cursor is located is not highlighted.

**Comment by snooppr** [comment]
1) I found what exactly causes the slowdown is the “slidebar”, which turns on automatically all the time; you can turn it off with the “b” key.  I believe that it is problematic and should be disabled by default (enabled from the menu or by key).
2) I could not return in version 3.0.2 the selection of the selected line under the cursor; in the xlsx format it does not work; in the csv format it selects the line, but not in all columns, but selectively.

v2.11.1
xlsx format, cursor on line "55-59", line highlighting is visible, in 3.0.2 it is not there.
![Screenshot_20240416-210415_Termux](https://github.com/saulpw/visidata/assets/61022210/5a8e4ed3-e1e4-4ccf-af93-6db5c4deeb6a)

**Comment by anjakefala** [comment]
Having an option that turns off sidebar by default will be tracked here: https://github.com/saulpw/visidata/issues/2202
